### ISSUE TITLE
GHC 8.6.3 boot libraries missing from Hackage

### ISSUE THREAD

**Issue by RyanGlScott** [haskell-infra/hackage-trustees#203]
The following boot libraries (which are bundled with GHC 8.6.3) appear to be missing from Hackage:

* [ ] `ghc-8.6.3`
* [ ] `ghc-boot-8.6.3`
* [ ] `ghc-boot-th-8.6.3`
* [ ] `ghc-heap-8.6.3`  (blocked on cabal bug)
* [ ] `ghci-8.6.3`
* [ ] `libiserv-8.6.3`

See also:

* #153
* #179
* #185
* #192

**Comment by RyanGlScott** [comment]
I hate to be a bother about this, but can something be done about this issue (and the litany of other similar issues)? It's extremely annoying not being able to look up the Haddocks for the latest `ghc` release online—I have to resort to Hackage mirrors in order to accomplish this.

Moreover, there's yet _another_ more recent release of `ghc` (8.6.4) than the one this issue is about. I'm hesitant to open another issue about it if it's just going to clog up the issue tracker unresolved, however.

If there's anything that someone like me can do to help move this along, please let me know. I'm keen to get a resolution on this.

**Comment by RyanGlScott** [comment]
I've just discovered that these libraries are on Stackage:

* [`ghc-8.6.3`](https://www.stackage.org/lts-13.5/package/ghc-8.6.3)
* [`ghc-boot-8.6.3`](https://www.stackage.org/lts-13.5/package/ghc-boot-8.6.3)
* [`ghc-boot-th-8.6.3`](https://www.stackage.org/lts-13.5/package/ghc-boot-th-8.6.3)
* [`ghc-heap-8.6.3`](https://www.stackage.org/lts-13.5/package/ghc-heap-8.6.3)
* [`ghci-8.6.3`](https://www.stackage.org/lts-13.5/package/ghci-8.6.3)

This means that I can at least find the Haddocks _somewhere_ online, which is the original reason I opened this issue. Closing.

**Comment by hvr** [comment]
> Moreover, there's yet another more recent release of ghc (8.6.4) than the one this issue is about. I'm hesitant to open another issue about it if it's just going to clog up the issue tracker unresolved, however.

For the record, I did actually finally got around to deal with those on Sunday evening (a day before you gave up...):

- http://hackage.haskell.org/package/ghc-8.6.4
- http://hackage.haskell.org/package/ghc-boot-8.6.4
- http://hackage.haskell.org/package/ghc-boot-th-8.6.4
- http://hackage.haskell.org/package/ghci-8.6.4

And I'll slowly work my way backwards to fill up the gaps
